week-4/InClass/B-objects-get-set/exercise-2.js

```diff
@@ -5,14 +5,15 @@
 */
 
 let phone = {
-    brand: 'iPhone,
-    model 'iPhone X'
+    brand: 'iPhone',
+    model :'iPhone X',

```

Comment:
Watch the space between the key/property and the colon here. This line should read `model: 'iPhone X',`